Zotero/Controllers/Database/Requests/StoreItemsDbResponseRequest.swift

```diff
@@ -191,6 +191,10 @@ struct StoreItemDbRequest: DbResponseRequest {
 
             case (FieldKeys.Item.note, _) where item.rawType == ItemTypes.note:
                 item.baseTitle = NotePreviewGenerator.preview(from: value) ?? value
+                item.htmlFreeContent = value.isEmpty ? nil : value
```

Comment:
Why is `value` not stripped of html tags as before?

Reply:
It's included in `NotePreviewGenerator.preview(from:)`, I just forgot about that initially.

Reply:
That is assigned to `item.baseTitle`, `value` is created in `let value = data.fields[keyPair] ?? ""`.
Is `NotePreviewGenerator.preview(from:)` used also before saving the initial `value` in the database?
What am I missing here?

Reply:
Oh you meant the `?? value` not `value` in `NotePreviewGenerator.preview(from: value)`? `NotePreviewGenerator.preview(from: value)` returns `nil` only when the string is empty, so I'll actually change this to `NotePreviewGenerator.preview(from: value) ?? ""` which makes more sense. And `NotePreviewGenerator` applies the stripping, so it should be applied to `htmlFreeContent`.